executeApiTests.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import pytest
+import threading
+
+from time import sleep
+from config.configManager import ConfigManager
+from utils.enum import Environment
+from utils import SetupCalls
+
+import config.globalvariables
+
+
+# File wide variables
+globalvars = config.globalvariables.GlobalVariables
+
+def _ParseArgs():
+    """
+    Parses the program arguments
+    Returns
+    -------
+    args
+    """
+
+    parser = argparse.ArgumentParser(
+        description='Input arguments',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+
+    parser.add_argument('-e', '--env',
+                        help='Environment used to check API of.',
+                        required=True,
+                        choices=["uat", "UAT", "ops", "OPS", "sit", "SIT", "ngap_uat", "ngap_ops", "ngap_sit"],
+                        metavar='ops, uat, sit')
+    parser.add_argument('-t', '--thread-count',
+                        help='How many thread to execute the tests simultaniously',
+                        required=False,
+                        type=int,
+                        default=1)
+   
+    args = parser.parse_args()
+    return args
+
+def _Run():
+    """
+    Run from command line.
+
+    Returns
+    -------
+    """
+
+    _args = _ParseArgs()
+    environment = Environment.from_str(_args.env)
+    threadCount = _args.thread_count
+
+    SetupCalls.SetUpLogging()
+    ConfigManager.InitializeConfig(
+        env = environment,
+        threadCount = threadCount)
+    argumentList = ["tests", "--junitxml=./test-results/report.xml", '-v']
+    # pytest.main(argumentList)
+    
+    # Set up paralel execution
```

Comment:
Do we still need this if locust is handling threading?

Reply:
I was planning to keep it until the test development as a fallback case if I encounter a situation that locust can't handle and I can't find a workaround for it.